docs/source/markdown/podman-build.1.md

```diff
@@ -99,7 +99,7 @@ different stages in COPY instruction.
 
 Valid values could be:
 
-* Local directory – e.g. --build-context project2=../path/to/project2/src
+* Local directory – e.g. --build-context project2=../path/to/project2/src (This option is not available with the remote Podman client. On Podman machine setup (i.e macOS and Winows) path must exists on the machine VM and not on the host)
```

Comment:
Remove "and not on the host", It can reside on both, usually the host and path in the VM are the same.